update organization machine role section to be clearer (#324)
bufbuild/docs.buf.build: docs/bsr/user-management.md

### Member {#org-member}

- Can view the organization and its members.

> - Derived the [base resource roles](#base-resource-roles) on existing organization

Reviewer: Maybe "Inherits" is better than "Derived" here?

Author: updated both to `Inherits` 👍

> - Have the [base resource role](#base-resource-roles) over the organization
>   existing resources, which defaults to [Write](#resource-write).

Reviewer 2: ```suggestion
- Has the [base resource permission](#base-resource-roles) on existing organization
 resources (the default is [Write](#resource-write)).
```

Author: actually I want to keep the word `role` as they are derived / inherited, let me adjust this a bit

### Machine {#org-machine}

- Can view the organization and its members.

> - Derived [Write](#resource-write) roles over existing organization resources, 

Reviewer: Is this still considered "derived" if it's "regardless of the base resource roles"? I think the previous wording is a little better, or @rhbuf's [suggestion](https://github.com/bufbuild/docs.buf.build/pull/324#discussion_r1067126174) of:

"Has [Write](#resource-write) permissions over existing organization resources,  regardless of the organization's [base resource roles](#base-resource-roles)."

Author: I tried to explicitly saying this is related to role, as `Can write to` might be ambiguous on if they can create resource (technically create resource is kind of a write action), and `write permission` feels similar to me

> - Have [Write](#resource-write) role over the organization existing resources, 
>   regardless of the organization's [base resource roles](#base-resource-roles).

Reviewer 2: ```suggestion
- Has [Write](#resource-write) permissions over existing organization resources, 
  regardless of the organization's [base resource roles](#base-resource-roles).
```
